### DISCUSSION TITLE
Making FastAPI deployable to Lambda out of the box.

### DISCUSSION THREAD

**Discussion by boscorona** [fastapi/fastapi#8901] [Questions]
Currently the process to deploy FastAPI to a AWS Lambda is extensive, I'd like to make it smooth and fluid especially with Serverless. I have a few ideas to make this possible. Curious to hear your thoughts?

There's the option of using Magnum but I think relying on another library to deploy this awesome async framework in a lambda should be simpler. Deploying with Serverless doesn't seem to be the most fluid experience right now.

**Comment by acidjunk** [comment]
It would be nice if it worked with Zappa. e.g. just 2 commands: `zappa init` and `zappa update`

FYI: Documented most of the process with mangum here: https://www.renedohmen.nl/deploy-fastapi-on-amazon-serverless/

Still a bit WIP.

**Comment by boscorona** [comment]
I'm thinking of adding a boilerplate to Serverless since that's the most popular, I deploy lambdas all the time, and always have to shim the deployment with Magnum.
